### PR TITLE
git-annotate, git-blame: add Ukrainian translation

### PR DESCRIPTION
**Pull request by burlakvo**
- [x] The page(s) are in the correct platform directories: `common`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages.uk/common/git-annotate.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# git annotate
+
+> Показує хеш коміту і останнього автора на кожному рядку у файлі.
+> Дивіться `git blame`, якому варто віддати перевагу над `git annotate`.
```
Comment (renie) [inline pages.uk/common/git-annotate.md:4]:
```suggestion
> Дивіться на `git blame`, якому варто віддати перевагу над `git annotate`.
```

**Review by renie — COMMENTED** [review]
With `на` sounds better to me. What do you think?
I am not a native speaker, so I prefer to not approve or block this PR for now.

File: pages.uk/common/git-annotate.md (hunk 1 above)
Comment (burlakvo) [reply pages.uk/common/git-annotate.md:4]: I've looked for any rules or at least examples of usage. Generally, it is used mostly with physical objects (e.g., Look at this car), but not with intangible ones (e.g., Watch a movie). Also, the English version of this doc used "See" and not "See at". So, I preferred to leave it as is.
But I caught there another issue :)

Anyway, thank you for your notice. I leave it unresolved while fixing that other place.

File: pages.uk/common/git-blame.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`git blame {{шлях/до/файлу}}`
+
+- Виводить файл з електронною поштою автора та хешем коміту на кожному рядку:
+
+`git blame {{-e|--show-email}} {{шлях/до/файлу}}`
+
+- Виводить файл лише із хешем коміту на кожному рядку:
+
+`git blame -s {{шлях/до/файлу}}`
```
[the remark this answers is not in the document]
Comment (burlakvo) [reply pages.uk/common/git-blame.md]: Changed

File: pages.uk/common/git-annotate.md (hunk 1 above)
Comment (burlakvo) [reply pages.uk/common/git-annotate.md]: Improved translations in other places.

**Review by renie — APPROVED** [review]
The new changes seems all ok to me.
But most of them are a gender matter. And I have never worked with ukrainian devs speaking in ukrainian. So I am not sure if they would refer to `git` itself, or its commands, in masculine, feminine or neuter. This waн I will trust your judgement here =).
Thank you very much for the contribution!  Слава :ukraine: !

**Comment by burlakvo** [comment]
Героям Слава!

Thanks for your comments @renie!

**Comment by renie** [comment]
No prob, @burlakvo! When I finish some work with pt_BR translations I am considering to give some attention to ukrainian translations also. Hopefully I will also see more translations of yours around here :smiley:

**Pull request merged**